contracts/ethereum/.solhint.json

```
@@ -0,0 +1,38 @@
+{
+  "extends": "solhint:recommended",
+  "rules": {
+    "func-visibility": ["error", {"ignoreConstructors": true}],
+    "state-visibility": "error",
+    "no-unused-vars": "error",
+    "const-name-snakecase": "error",
+    "contract-name-camelcase": "error",
+    "event-name-camelcase": "error",
+    "func-name-mixedcase": "error",
+    "modifier-name-mixedcase": "error",
```

Biome (2.1.2)

[error] The key modifier-name-mixedcase was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)